packages/react-components/react-components/src/Concepts/Icons/Icon.stories.mdx

```diff
@@ -25,7 +25,60 @@ You can import the sized icons in a similar way:
 import { AccessTime24Filled } from '@fluentui/react-icons';
 ```
 
+### List of available props
+
 Each icon also accepts `className` and `primaryFill` props for styling.
+You can also use the `bundleIcon` method to bundle a `filled` and unfilled version of an icon and this will combine them into one icon.
+
+| - Name -    | Description                                                                                                            | Default        |
+| ----------- | ---------------------------------------------------------------------------------------------------------------------- | -------------- |
+| className   | Used to style the icon                                                                                                 | -              |
+| filled      | Used to determine whether the filled or unfilled version of an icon is the default when `bundleIcon` is used `boolean` | -              |
+| primaryFill | Used to change the primary fill of the icon                                                                            | `currentColor` |
+| title       | Specifies the title attribute for the svg                                                                              | -              |
+
+### Example
+
+```tsx
```

Comment:
This example immediately jumps to the `bundleIcon` usage. There is no explanation what this example is doing and why `bundleIcon` is necessary here.

Please add some description about the problem being solved here, which is switching between fill and outline icon variants on hover